src/components/CRAndTDLSidePanel/index.tsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Grid, GridContainer, Icon } from '@trussworks/react-uswds';
+import i18n from 'config/i18n';
+
+import ExternalLink from 'components/ExternalLink';
+import TruncatedText from 'components/TruncatedText';
+
+import './index.scss';
+
+export type CRAndTDLSidePanelProps = {
+  isCR: boolean;
+  id: string;
+  title?: string | null;
+  crStatus?: string | null;
+  emergencyCrFlag?: boolean | null;
+  implementationDate?: string | null;
+  issuedDate?: string | null;
+  sensitiveFlag?: boolean | null;
+  initiator?: string | null;
+  relatedCrTdlNumbers?: string | null;
+  crSummary?: {
+    rawContent: string;
+  } | null;
+};
+
+const CRAndTDLSidePanel = ({
+  isCR,
+  id,
+  title,
+  crStatus,
+  emergencyCrFlag,
+  implementationDate,
+  issuedDate,
+  sensitiveFlag,
+  initiator,
+  relatedCrTdlNumbers,
+  crSummary
+}: CRAndTDLSidePanelProps) => {
+  const { t: crtdlsT } = useTranslation('crtdlsMisc');
+
+  const properlyCapitalizeInitiator = (fullName: string) => {
+    const [namePart, parenthesisPart] = fullName.split(' (');
+
+    const capitalizedName = namePart
+      .toLowerCase()
+      .replace(/\b\w/g, char => char.toUpperCase());
+
+    return parenthesisPart
+      ? `${capitalizedName} (${parenthesisPart}`
+      : capitalizedName;
+  };
```

Comment:
Can probably move this out of the component itself, maybe in a util or still in this file is fine, otherwise it gets created on every render.  This function could also maybe use a test for its logic.